CMakeLists.txt

```diff
@@ -135,3 +135,10 @@ endif()
 # ------ Extra scripts to install
 file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/PythonScriptStore/RegressionTest/regression_test_base_tool.py
     DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/PythonScriptStore/RegressionTest)
+
```

Comment:
these are not necessary as the building arguments have already been done with the cmakelist in the root.